src/main/preprocessing/activity_tracker_handler.py

```diff
@@ -188,8 +190,12 @@ def get_files_from_ati(activity_tracker_data: pd.DataFrame) -> List[str]:
     paths_dict = {}
     for current_path in paths:
         path = get_parent_folder(current_path)
-        file = get_name_from_path(current_path)
-        if paths_dict.get(file) is None:
+        try:
+            file = get_name_from_path(current_path)
+        except ValueError:
+            # If the file has an invalid extension, we should miss it
```

Comment:
I tested it with the new data and one ide-events file contains a file with name: __init__.pyi. I don't know, why a user created it, but I think this can be a common mistake and we should skip such files since we only want to process еру task files, and they have the correct extension. What do you think?

Reply:
Oh, okay, thanks for explaining